examples/helloworld/BUILD

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cc_binary(
+    name = "helloworld",
+    srcs = ["helloworld.cc"],
+    deps = [
+        "@io_opentelemetry_cpp//api/include/opentelemetry/trace",
```

Comment:
I'm not happy with this. There should be a single dep and it should look different / shorter.

Reply:
Is it possible for local builds make it **@io_opentelemetry_cpp/include**

Then that include directory contains:
```
opentelemetry
opentelemetry/sdk
```

User code consuming the SDK has to use it as:
```
#include "opentelemetry/trace/trace.hpp"
#include "opentelemetry/sdk/whatever/custom.hpp"
```

And for other build systems that may be consuming the SDK from /usr/include, the layout is gonna be:
```
/usr/local/include/opentelemetry/ <-- API here
/usr/local/include/opentelemetry/sdk/ <-- other 'advanced' stuff here
```
requiring the customer to deploy libopentelemetry-dev (or libotel-dev?) using their preferred packaging system in their distribution (dpkg, brew, chocolatey, nuget, vcpkg, etc.).

Reply:
Per #13 it's `@io_opentelemetry_cpp//api`

In future (not this PR) we'll also have `//sdk`

The includes work like you said.